### PR TITLE
Add `keyboard_teleoperation.launch` to `husky_control` package

### PR DESCRIPTION
**Pull request by Tinker-Twins**
Added `teleop_keyboard.launch` file to `husky_control` package for enabling convenient teleoperation of Husky robot using standard computer keyboard.

We believe that this would be a valuable addition to the `husky_control` package since:
- Not everyone has access to a joystick controller (but a computer does come with a keyboard)
- Beginners (especially) would feel at ease given that a "ready-to-use" launch file is available with Husky packages

P.S. with respect to PR #242, the updated `teleop_keyboard.launch` file now makes use of the [`teleop_twist_keyboard`](https://github.com/ros-teleop/teleop_twist_keyboard) package and the custom `teleop_keyboard.py` script has been removed from `husky_control` package.

### CODE REVIEW

File: husky_control/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Changelog for package husky_control
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.6.5 (2022-09-26)
+------------------
+* Add keyboard teleoperation functionality.
+* Contributors: Tinker Twins (Chinmay Samak & Tanmay Samak)
+
```
Comment (civerachb-cpr) [inline husky_control/CHANGELOG.rst]: Please do not edit the changelog directly; this gets automatically updated when the package is released

File: husky_control/package.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>husky_control</name>
-  <version>0.6.4</version>
+  <version>0.6.5</version>
```
Comment (civerachb-cpr) [inline husky_control/package.xml]: Please do not edit the package version; this gets updated when the package is released

File: husky_gazebo/launch/realsense.launch [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<?xml version="1.0"?>
```
Comment (civerachb-cpr) [inline husky_gazebo/launch/realsense.launch]: This is a separate fix and not directly related to this change

**Review by civerachb-cpr — CHANGES_REQUESTED** [review]

File: husky_control/CHANGELOG.rst (hunk 1 above)
Comment (Tinker-Twins) [reply husky_control/CHANGELOG.rst]: Sorry, did not know that this gets updated automatically when the package is released. The requested changes have been made.

File: husky_control/package.xml (hunk 2 above)
Comment (Tinker-Twins) [reply husky_control/package.xml]: Sorry, did not know that this gets updated automatically when the package is released. The requested changes have been made.

File: husky_gazebo/launch/realsense.launch (hunk 3 above)
Comment (Tinker-Twins) [reply husky_gazebo/launch/realsense.launch]: You're right, reverted back (the requested changes have been made). Can create a separate PR for this fix.

**Comment by civerachb-cpr** [comment]
Everything looks OK to me now (thanks for those additional changes).  @tonybaltovski, the maintainer of these packages, is away at the moment, so please be patient until he's back and can give his final approval.

**Comment by Tinker-Twins** [comment]
@civerachb-cpr or @tonybaltovski could you please review the changes?

**Comment by civerachb-cpr** [comment]
Tony's still away. Everything looks OK to me, so I'll merge this.  Thanks for your patience!

**Pull request merged**